vit_vqgan/modeling_vit_vqgan.py

```diff
@@ -470,14 +465,17 @@ def setup(self):
             self.to_patches = FeedForwardLayer(
                 dim1=self.config.intermediate_size,
                 dim2=input_dim,
-                activation="tanh",
+                activation=self.config.hidden_act,
                 config=self.config,
                 dtype=self.dtype,
             )
+        self.encoder_ln = nn.LayerNorm(epsilon=self.config.layer_norm_eps, dtype=self.dtype, use_scale=False)
```

Comment:
We are calling `self.encoder`, in multiple methods (`self.encode` and `__call__`).
Let's move the layernorm in the encoder class, so it always get's applied. Right now it's missing from call.

Reply:
Should we also move `factor_in` in encoder (and `factor_out` in decoder)?
I think that the `LayerNorm` should be applied after the projection (that way it also does not need scaling)

Reply:
I see. So let's leave it here then.

Reply:
Not sure about the layernorm placement. I think either should work, but let's go with this.

Reply:
(in the conv vae normalization happens before  the projection.)